android/src/main/java/com/pspdfkit/react/PSPDFKitModule.java

```diff
@@ -3,7 +3,7 @@
  *
  *   PSPDFKit
  *
- *   Copyright © 2017 PSPDFKit GmbH. All rights reserved.
```

Comment:
Do we have more files with outdated copyright header? We try to keep the original year in here. So it should be updated to `2017-2018`.